core/src/main/java/io/questdb/cairo/sql/PageFrameMemoryRecord.java

```diff
@@ -48,16 +50,18 @@
 import io.questdb.std.str.Utf8SplitString;
 import org.jetbrains.annotations.Nullable;
 
-import java.io.Closeable;
-
 /**
- * Must be initialized with a {@link #init(PageFrameMemory)} call
- * for a given page frame before any use.
+ * Must be initialized with a {@link PageFrameMemoryPool#navigateTo(int, PageFrameMemoryRecord)}
+ * or {@link #init(PageFrameMemory)} call for a given page frame before any use.
  */
-public class PageFrameMemoryRecord implements Record, StableStringSource, Closeable {
+public class PageFrameMemoryRecord implements Record, StableStringSource, QuietCloseable, Mutable {
+    // Letters are used to track usage in PageFrameMemoryPool
+    public static final byte RECORD_A_LETTER = 0;
+    public static final byte RECORD_B_LETTER = 1;
     private final ObjList<MemoryCR.ByteSequenceView> bsViews = new ObjList<>();
     private final ObjList<DirectString> csViewsA = new ObjList<>();
     private final ObjList<DirectString> csViewsB = new ObjList<>();
+    private final byte letter; // 0 means A, 1 means B
```

Comment:
can we have a bit of commentary around meaning of the letter?

Reply:
Expanded the comment in bff3be12fc4b4bd622676d07b92abeb45962585b